index.bs

```diff
@@ -1072,12 +1130,31 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
         significant.
       </p>
 
+      <p>
+        HDR detection is more nuanced. Adding colorGamut, transferFunction, and
+        hdrMetadataType has the potential to add significant entropy. However,
+        for UAs whose decoders are implemented in software and therefore whose
+        capabilities are fixed across devices, this feature adds no effective
+        entropy. Additionally, for many cases, devices tend to fall into large
+        categories, within which capabilities are similar thus minimizing
+        effective entropy.
+      </p>
+
+      <p>
+        Because these features require directly calling asynchronous Web APIs,
```

Comment:
This p could go. Its technically correct, and all the remaining things are async, but we recently concluded that sync and async are both equally detectable.